test/Compatibility/attr_usableFromInline_protocol.swift

```diff
@@ -7,8 +7,8 @@ public protocol PublicProtoWithReqs {
 }
 
 @usableFromInline struct UFIAdopter<T> : PublicProtoWithReqs {}
-// expected-warning@-1 {{type alias 'Assoc' should be declared '@usableFromInline' because it matches a requirement in protocol 'PublicProtoWithReqs'}} {{none}}
-// expected-warning@-2 {{instance method 'foo()' should be declared '@usableFromInline' because it matches a requirement in protocol 'PublicProtoWithReqs'}} {{none}}
+// expected-warning@-1 {{type alias 'Assoc' must be declared '@usableFromInline' because it matches a requirement in protocol 'PublicProtoWithReqs'; this is an error in Swift 5}} {{none}}
```

Comment:
Looks probably intentional and fine to me but I do want to flag this just in case - are the wording changes here a drive by improvement coming from making other changes in the area?

Reply:
Yes. When I was updating this diagnostic, I noticed that it had a separate `warn_` variant because it predated the introduction of `warnUntilSwiftVersion()`.